lib/rollbar/middleware/js.rb

```diff
@@ -133,6 +133,7 @@ def html_safe_if_needed(string)
       def append_nonce?
         defined?(::SecureHeaders) && ::SecureHeaders.respond_to?(:content_security_policy_script_nonce) &&
           defined?(::SecureHeaders::Configuration) &&
+          !::SecureHeaders::Configuration.get.csp.opt_out? &&
```

Comment:
The similar check on 2.x would be `SecureHeaders::Configuration.csp.nil?`. Version detection is necessary, alas, from what I've gathered.